compiler/backend/source_to_modScript.sml

```diff
@@ -15,8 +15,12 @@ val _ = new_theory"source_to_mod";
  *)
 
 val Bool_def = Define `
- Bool b = (App (Opb (if b then Leq else Lt)) [Lit (IntLit 0); Lit (IntLit 0)])`;
+ Bool t b = (App t (Opb (if b then Leq else Lt)) [Lit t (IntLit 0); Lit t (IntLit 0)])`;
```

Comment:
We might need to clarify what `t` refers to here. The variable naming is very opaque, and the fact that we are introducing an entirely new parameter will most liktely add significant amounts of confusion for someone not very familiar with what we are trying to accomplish (namely, adding position information).